.github/workflows/ci.yml

```diff
@@ -3,6 +3,16 @@ name: Continuous Integration
 on: [push, pull_request]
 
 jobs:
+  license:
+    name: Check source files for license headers
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Check for license headers
+        run: bash check_license.sh
```

Comment:
Can we move this script to a `bin` directory? That way the project's root won't have `.sh` scripts if we decide to create new ones.

Reply:
Done!